lib/vast/Dialect/HighLevel/HighLevelOps.cpp

```diff
@@ -202,6 +202,11 @@ namespace vast::hl
         InsertionGuard guard(bld);
         build_region(bld, st, constants);
     }
+    void EnumDeclOp::build(Builder &bld, State &st, llvm::StringRef name) {
+        st.addAttribute("name", bld.getStringAttr(name));
+        auto reg = build_region(bld, st, {});
```

Comment:
add `build_empty_region` which can be used on other places where `emplaceBlock` is called